projects/isy-angular-widgets/assets/theme/bootstrap4/bootstrap4-light/blue/_variables.scss

```diff
@@ -1,12 +1,12 @@
 $primaryLightColor: rgba(38, 143, 255, 0.5) !default;
-$primaryColor: #45484d !default;
-$primaryDarkColor: #89909a !default;
```

Comment:
Warum gibt es Änderungen in dieser Datei? Das sollte m.E. über das IsyFact Themen überschrieben werden